features/issue/components/issuesList/IssuesList.tsx

```diff
@@ -7,12 +7,27 @@ import { useToast } from '@/components/ui/use-toast'
 import { useGetIssues } from '../../api'
 import { LoadingSpinner } from '@/components/ui/LoadingSpinner'
 import { Searchbar, StatusFilter } from './issuesFilter'
+import { IssuesPagination } from './IssuesPagination'
 
 export const IssuesList = () => {
 	const { data: issues, isLoading, isError } = useGetIssues()
 	const { toast } = useToast()
 	const [search, setSearch] = useState('')
 	const [statusFilter, setStatusFilter] = useState('')
+	const [currentPage, setCurrentPage] = useState(1)
+
+	const filteredIssues = issues.filter(issue => {
+		return (
+			issue.title.toLowerCase().includes(search.toLowerCase()) &&
+			issue.status.includes(statusFilter)
+		)
```

Comment:
Paginacja po stronie klienta, jest generalnie średnim pomysłem. Paginacja ma na celu 2 rzeczy: a) Nie wyświetlać w huj danych userowi, b) Żeby api nie zwracało w huj danych. Client side paginacja pomaga tylko w warunku a), backend cały czas zwraca wszystkie dostępne issue.

To samo tyczy się filtrów które zrobiłeś. Generalnie przy małej aplikacji gdzie tych rekordów nie jest huj wie ile to nie jest problem, wręcz fajniej to wygląda jak jest client side bo wszystko działa instant, ale w poważnych apkach tak się nie robi